packages/react/src/Button/Button.dev.stories.tsx

```diff
@@ -53,20 +53,17 @@ export const TestSxProp = () => {
           },
           [`@media screen and (max-width: 768px)`]: {
             color: 'maroon',
-          },
-          '@media screen and (min-width: 768px)': {
```

Comment:
I understand the intent of these examples is to show that consumers can customize their buttons via `sx` prop. However, I also don't think this (customizing focus) is good practice we want to endorse. It is easier for me to just remove the example.